R/check.R

```diff
@@ -136,19 +136,19 @@ verifyTests <- function(acceptedWarnings) {
 #'
 #' Run linter and stop on linter warning unless linter warnings are allowed.
 #'
-#' @param allowLinterWarnings If FALSE (the default) will stop on linter warnings.
+#' @param allowLinterWarnings If FALSE will stop on linter warnings.
 #'
 #' @author Pascal Sauer
 #' @export
-verifyLinter <- function(allowLinterWarnings = FALSE) {
+verifyLinter <- function(allowLinterWarnings = TRUE) {
```

Comment:
Changing the default value does not seem necessary here. Technically changing default values is an API/interface breaking change; there might be a script somewhere on a local machine that relies on this default value (admittedly quite unlikely here). I also think the default should be to not allow linter warnings :)
```suggestion
verifyLinter <- function(allowLinterWarnings = FALSE) {
```

Reply:
I applied the suggestion now